### PR TITLE
T-92 Аdd slider to the modal playgrounds details

### PR DESCRIPTION
**Pull request by Bohdan-R**

### CODE REVIEW

File: components/PlaygroundModalContent/PlaygroundModalContent.jsx [hunk 1]
```diff
@@ -21,12 +22,30 @@ const PlaygroundModalContent = ({ playground }) => {
...
-        <Image
-          className={styles.bgImage}
-          src={playground.img}
-          alt=""
-          layout="responsive"
-        />
+        <Slider
+          slidesToShow={1}
+          slidesToScroll={1}
+          isArrows={true}
```
Comment (WebTarantul) [inline components/PlaygroundModalContent/PlaygroundModalContent.jsx:28]: I think 'withArrows' is more clear name.

File: components/PlaygroundModalContent/PlaygroundModalContent.jsx [hunk 2]
```diff
@@ -21,12 +22,30 @@ const PlaygroundModalContent = ({ playground }) => {
...
+          isModal={true}
+          classNameBox={styles.sliderBox}
+          classNameDots={styles.dots}
+          classNameDotsModal={styles.modalDots}
+          isArrowColorBlack={false}
+          arrayLength={playground.images.length}
+        >
+          {playground.images.map((img, i) => (
+            <>
```
Comment (WebTarantul) [inline components/PlaygroundModalContent/PlaygroundModalContent.jsx:37]: Is no reason to use Fragment here

File: components/PlaygroundModalContent/PlaygroundModalContent.module.scss [hunk 3]
```diff
@@ -67,3 +76,45 @@
     margin: 30px 0 36px;
   }
 }
+.sliderBox .dots.modalDots {
```
Comment (WebTarantul) [inline components/PlaygroundModalContent/PlaygroundModalContent.module.scss:79]: Here is no reason to use too many css classes. You need as many classes as you need to redefine styles.

File: components/PlaygroundModalContent/PlaygroundModalContent.module.scss [hunk 4]
```diff
@@ -67,3 +76,45 @@
...
+  @media screen and (max-width: 950px) {
+    position: absolute;
+    bottom: -18px;
+    height: 5px;
+  }
+}
+.sliderBox .dots.modalDots ul {
```
Comment (WebTarantul) [inline components/PlaygroundModalContent/PlaygroundModalContent.module.scss:86]: Here is no reason to use too many css classes. You need as many classes as you need to redefine styles.


Plese change all lines according that tip

File: components/Slider/Slider.js [hunk 5]
```diff
@@ -31,6 +31,7 @@ const SlickSlider = ({
   classNameBox,
   classNameArrow,
```
Comment (WebTarantul) [inline components/Slider/Slider.js:32]: I think you don't need too many props for css class. You can use one class for wrapper element and then use other exists classes of inside elements

**Review by WebTarantul — CHANGES_REQUESTED** [review]

**Comment by Dimirolz** [comment]
@WebTarantul @IgVol дайте ваше ревью, как будет время, Богдан сделал необходимые фиксы

**Comment by Dimirolz** [comment]
@Bohdan-R пофикси конфликты, что бы я смог смерджить

**Review by Dimirolz — APPROVED** [review]

**Pull request merged**